packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -185,41 +191,65 @@ export default class OperatorModeStackItem extends Component<Signature> {
         >
           <:actions>
             {{#if (eq @item.format 'isolated')}}
-              <IconButton
-                @icon='icon-pencil'
-                @width='24px'
-                @height='24px'
-                @tooltip='Edit'
-                class='icon-button'
-                aria-label='Edit'
-                {{on 'click' (fn @edit @item)}}
-                data-test-edit-button
-              />
+              <Tooltip
+                @placement='top'
+                @offset={{6}}>
```

Comment:
`@offset={{6}}>` is repeated so consider using that as default in the tooltip component